server/rest/src/main/java/org/infinispan/rest/server/CacheControl.java

```diff
@@ -0,0 +1,26 @@
+package org.infinispan.rest.server;
+
+public class CacheControl {
+
+   private static final String NO_CACHE = "no-cache";
+   private static final String MAX_AGE = "max-age";
+
+   private String cacheControl;
+
+   private CacheControl(String cacheControl) {
+      this.cacheControl = cacheControl;
+   }
+
+   public static CacheControl noCache() {
+      return new CacheControl(NO_CACHE);
```

Comment:
I meant to cache the actual `CacheControl` object so we don't have to create a new one each time.